lib/ui/platform.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of dart_ui;
+
+/// Platform services provided to the application by Flutter.
+///
+/// Invoke a platform service by passing a request as json.
+
```

Comment:
The way these `///` comments works is that you have to attach them to symbol for them to show up on the web site.